pkg/support/dump.go

```diff
@@ -104,10 +110,44 @@ func dumpNetworkPolicyResources(fs afero.Fs, executor exec.Interface, basedir st
 	return dumpAntctlGet(fs, executor, "addressgroups", basedir)
 }
 
-// fileCopy copies files under the srcDir to the targetDir. Only files whose
-// name matches the prefixFilter will be copied. Copied files will be located
-// under the same relative path.
-func fileCopy(fs afero.Fs, targetDir string, srcDir string, prefixFilter string) error {
+func timestampFilter(since string) *time.Time {
+	var timeFilter *time.Time
+	if since != "" {
+		duration, _ := str2duration.ParseDuration(since)
```

Comment:
```suggestion
		duration, _ := time.ParseDuration(since)
```
The new dependencies are not necessary.

Reply:
The initial thought was trying to support `d` as time duration unit. Since it's not a mandatory request and can be represents by `h`, `time.ParseDuration` should works fine too. Thanks!

Reply:
`kubectl logs` used same function to parse "--since" and I just saw its biggest unit is also `h`. If no one complains about it, we should be fine too.

Reply:
Updated